src/components/Stakeholders/Add.js

```diff
@@ -0,0 +1,354 @@
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                error={!!errors?.phone}
+                helperText={errors.phone}
+                className={classes.textField}
+                label="Phone"
+                variant="outlined"
+                name="phone"
+                type="number"
+                onChange={handleChange}
+                value={data.phone}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                className={classes.textField}
+                label="Website"
+                variant="outlined"
+                name="website"
+                onChange={handleChange}
+                value={data.website}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                className={classes.textField}
+                label="Logo Url"
+                variant="outlined"
+                name="logo_url"
+                onChange={handleChange}
+                value={data.logo_url}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <TextField
+                className={classes.textField}
+                label="Map"
+                variant="outlined"
+                name="map"
+                placeholder="map_address"
+                onChange={handleChange}
+                value={data.map}
+                onKeyDown={handleEnterPress}
+              />
+            </FormControl>
+          </Grid>
+          <Grid container justify-content="space-between">
+            <FormControl xs={12} sm={6} className={classes.root}>
+              <FormLabel id="relation">Relationship</FormLabel>
+              <RadioGroup
+                aria-labelledby="radio-buttons-group-label"
+                name="relation"
+                value={data.relation}
+                className={classes.radioGroup}
+                onKeyDown={handleEnterPress}
+              >
+                <FormControlLabel
+                  control={
+                    <Radio
+                      label="Parent"
+                      value="parents"
+                      onChange={handleChange}
+                    />
+                  }
+                  label="Parent"
+                />
+                <FormControlLabel
+                  label="Child"
+                  value="children"
+                  control={<Radio />}
+                  onChange={handleChange}
+                />
+              </RadioGroup>
+              <TextField
+                className={classes.textField}
+                label="Relation ID"
```

Comment:
Is this meant to be the ID for the related stakeholder? If so, this might be more useable as a dropdown.

Reply:
Ok, good idea